### DISCUSSION TITLE
Live / motion photos?

### DISCUSSION THREAD

**Discussion by jschwalbe** [damongolding/immich-kiosk#43] [Ideas]
Just throwing this out there, would there be a possibility to show the videos in motion? (on iOS I believe they're called Live Photos, not sure about android, maybe Motion photos?) I realize you've just had a baby, and congrats! So no rush on this request, but wondering how hard it would be to implement. Thanks!

**Comment by SelfSufficient08** [comment]
This would be cool to play videos under a certain length.  Like anything less than 5 or 10 seconds.  I saw one of those picture frames at a friends house that did this with all the iphone photos.  Was really cool to relive short moments.  If possible, maybe a variable that lets the user choose the max video length.

**Comment by damongolding** [comment]
I am looking into adding video support. It's a rather large bit of work so might take some time and will come with limitations, like no audio due to autoplay restrictions

**Reply by SelfSufficient08** [reply]
You're awesome man.  I personally wouldn't want audio, just the video playing during the time specified for regular photo swaps.

**Comment by damongolding** [comment]
I've started to support video in v0.16.0 @jschwalbe @SelfSufficient08. Limited to albums for now.

**Reply by austinliuu** [reply]
Are live photo/motion photos considered as video? I searched in my immich library and these photos did not appear with 'video' filter
![image](https://github.com/user-attachments/assets/7e50cb89-1c9c-4fb0-bf77-0bdce4f26317)
